src/pudl/helpers.py

```diff
@@ -2344,15 +2344,11 @@ class ParquetData(BaseModel):
 
     table_name: str
     partitions: dict[str, Any] = {}
-    use_output_dir: bool = False
 
     @property
     def parquet_directory(self) -> Path:
         """Get path to directory for writing/reading parquet files."""
-        if self.use_output_dir:
-            parquet_path = PudlPaths().parquet_path() / self.table_name
-        else:
-            parquet_path = PudlPaths().parquet_transform_dir / self.table_name
+        parquet_path = PudlPaths().parquet_path() / self.table_name
```

Comment:
Just write intermediate files to output directory. These will always be contained within a sub-directory, so they shouldn't be distributed.